OpenRA.Game/Graphics/WorldRenderer.cs

```diff
@@ -186,6 +213,15 @@ public void Draw()
 
 			Game.Renderer.Flush();
 
+			if (enableDepthBuffer)
+				Game.Renderer.ClearDepthBuffer();
+
+			for (var i = 0; i < underRenderables.Count; i++)
+				underRenderables[i].Render(this);
+
+			if (enableDepthBuffer)
+				Game.Renderer.ClearDepthBuffer();
```

Comment:
Clearing the depth buffer between the terrain and other world renderable breaks behind-cliff rendering in TS and other depth-enabled mods.

Reply:
It should be possible to disable the depth test while drawing the lines without touching the depth buffer, but this will require plumbing changes down into the core GL code.